package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "express-fileupload",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
new option is a feature — it should have been minor release (1.6.0)

Reply:
Hi @RobinTail 

Partially agree with u.
In this case decided not to change 5 to 6, since change quite small and 100% backward compatible.

Reply:
If it was incompatible, then it would be 2.0.0

https://semver.org

FYI